### PR TITLE
Catch runtime exception

### PR DESCRIPTION
**Pull request by stoeffel**

### CODE REVIEW

**Comment by rtfeldman** [comment]
Thanks @stoeffel!

File: lib/supervisor.js [hunk 1]
```diff
@@ -54,20 +54,32 @@ function run(
...
 
   var workers = _.range(0, processes).map(function(index) {
     var worker = child_process.fork(dest);
 
-    if (watch && !isMachineReadable) {
-      worker.on("close", function(code, signal) {
+    worker.on("close", function(code, signal) {
+      if (watch && !isMachineReadable) {
+        if (code !== 0) {
+          reportRuntimeException();
```
Comment (stoeffel) [inline lib/supervisor.js]: @rtfeldman this shows up during `watch` if you have a failing test 😢

**Comment by rtfeldman** [comment]
Bah...want to make a quick PR?

**Comment by stoeffel** [comment]
@rtfeldman sorry, can't today. Maybe someone else wants to volunteer?

**Pull request merged**